server/functions.lua

```diff
@@ -245,7 +238,9 @@ function QBCore.Functions.CreateVehicle(source, model, coords, warp)
     local veh = CreateVehicleServerSetter(model, vehicleType, coords.x, coords.y, coords.z, coords.w)
     while not DoesEntityExist(veh) do Wait(0) end
     if warp then TaskWarpPedIntoVehicle(GetPlayerPed(source), veh, -1) end
-    return veh
+    local state = Entity(veh).state
+    state:set('initVehicle', true, true)
```

Comment:
```suggestion
    Entity(veh).state:set('initVehicle', true, true)
```

We do not need state localized because we do not index it multiple times